### PR TITLE
[fs-209]: Creating A Family Should Redirect To Detail Page

### PR DESCRIPTION
**Pull request by erfan-goodarzi**
This PR adds navigating to the detail page after the family is created successfully.

### CODE REVIEW

File: app/components/CreateFamily/CreateFamilyForm/CreateFamilyForm.tsx [hunk 1]
```diff
@@ -2,8 +2,10 @@ import { useCreateFamilyMutation } from '@camp/data-layer';
 import { showNotification } from '@camp/design';
 import { createResolver, familySchema } from '@camp/domain';
 import { messages } from '@camp/messages';
+import type { AppRoute } from '@camp/router';
 import { createTestAttr } from '@camp/test';
 import { Button, Group, Stack, TextInput } from '@mantine/core';
+import { useNavigate } from '@tanstack/react-location';
```
Comment (ASafaeirad) [inline app/components/CreateFamily/CreateFamilyForm/CreateFamilyForm.tsx:8]: Can we you our router instead?

File: cypress.config.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ export default defineConfig({
       APP_AUTH0_CLIENT_ID: getEnv('APP_AUTH0_CLIENT_ID'),
       APP_API_ENDPOINT: getEnv('APP_API_ENDPOINT'),
       APP_HASURA_ADMIN_SECRET: getEnv('APP_HASURA_ADMIN_SECRET'),
+      http_proxy: getEnv('HTTP_PROXY'),
```
Comment (ASafaeirad) [inline cypress.config.ts:25]: Is it used?

Comment (erfan-goodarzi) [reply cypress.config.ts:25]: Yeah, I had an issue with that and I needed to add it for e2e to work for me.

Comment (ASafaeirad) [reply cypress.config.ts:25]: But I didn't find it in the code

**Review by ASafaeirad — APPROVED** [review]

**Pull request merged**